tests/sdk/v2/driver/resp_mocks.py

```diff
@@ -240,7 +240,19 @@ def make_get_wf_run_results_response():
     }
 
 
-def make_get_wf_run_result_response(result_obj: Any):
+def make_get_wf_run_result_response(result_list: List[Any]):
+    """
+    Based on:
+        https://github.com/zapatacomputing/workflow-driver/blob/main/openapi/src/resources/run-result.yaml#L13
+    """
+
+    return [
+        result_from_artifact(result_obj, ArtifactFormat.AUTO).dict()
+        for result_obj in result_list
+    ]
+
+
+def make_get_wf_run_result_legacy_response(result_obj: Any):
```

Comment:
**suggestion**: I thought I got it but I don't. It's very easy to make a mistake when reasoning about these responses, esp. because there seem to be multiple levels of nesting and serialization. For the sake of stability of regression tests, could you please add test examples with explicit response bodies we expect from CE? For example:

`tests/sdk/v2/driver/responses/v_46_0_workflow_single_output.json`:
```
{
  "data": { ... }
}
```

`tests/sdk/v2/driver/responses/v_46_0_workflow_multi_output.json`:
```
{
  "data": { ... }
}
```
etc